.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: node_js
-sudo: false
```

Comment:
Why was this removed?

Reply:
AFAIU this effectively became a noop as of https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517